drizzlepac/hapmultisequencer.py

```diff
@@ -70,7 +70,6 @@
 log = logutil.create_logger(__name__, level=logutil.logging.NOTSET, stream=sys.stdout,
                             format=SPLUNK_MSG_FORMAT, datefmt=MSG_DATEFMT)
 __version__ = 0.1
```

Comment:
`__version__` should be string and maybe removed altogether from individual files? I thought this was one of the points of this PR.

Reply:
And this comment seems to apply to all HAP files